css/styles.css

```diff
@@ -0,0 +1,127 @@
+
+.containertwo {
+  background-color: #562B81;
+  height: 50px;
+  padding: 1px;
+  display: flex;
+}
+.heading-text {
+  margin: auto;
+  flex-basis: 43%;
+  color: white;
+  font-size: 20px;
+  font-family: Proza Libre;
+  font-weight: 100;
+  background-color: #562B81;
+}
+
+p {
+  font-family: sans-serif;
+}
+
+.new-entry {
+  font-family: sans-serif;
+  color: rgb(91, 89, 89);
+  font-size: 23px;
+}
+
+img:nth-child(1) {
+  height: 300px;
+  width: 300px;
+}
+
+.titletext {
+  height: 2rem;
+  border-radius: .2rem;
+  border: solid 1px rgb(174, 173, 173);
+  margin-right: 2rem;
+}
+.title-text:focus, .photo-text:focus,
+.textarea-1:focus {
+  outline: none;
+  border: 2px solid #562B81;
+}
+
+
+.phototext {
+  height: 2rem;
+  border-radius: .2rem;
+  border: solid 1px rgb(174, 173, 173);
+  margin-right: 2rem;
+}
+
+.notestext {
+  height: 5rem;
+  border-radius: .2rem;
+  border: solid 1px rgb(174, 173, 173);
+  margin-right: 2rem;
+  padding-bottom: 3.2rem;
+  padding-left: .5rem;
+}
+
+.column-half {
```

Comment:
the layout classes are in `layout.css` and should not be changed. also, you don't want t a column half class set to a width of 6.7ish percent :)